index.html

```diff
@@ -7,32 +8,34 @@
 </head>
 <body>
 
-<pre id="paste">
-                         _
-     ___ ___ _____ _____|_|___  <span class="sickle">☭</span>
-    |  _| . |     |     | | -_|
-    |___|___|_|_|_|_|_|_|_|___|
-
-    commie is a pastebin script with line commenting support.
-    available at <a href="https://github.com/splitbrain/commie">github</a>. made by <a href="http://www.splitbrain.org">splitbrain</a>.
-
+<div id="content">
 
+<pre>
+  _____                 _
+ |     |___ _____ _____|_|___  <span class="sickle">☭</span>
+ |   --| . |     |     | | -_|
+ |_____|___|_|_|_|_|_|_|_|___|
+</pre>
 
+<div id="paste">
```

Comment:
Why switching from the semantic ```pre``` (which signal preformatted text) to the unspecific ```div``` here?